### PR TITLE
Member details updates

### PR DESCRIPTION
**Pull request by PottertheJavaOtter**
This updates every the `About -> Meet the team` page as well as the bio pages for team members to the most recent version on marvel.
This is up to date with all the bio changes from May as well. 
https://openforge.teamwork.com/#/tasks/16775297

### CODE REVIEW

**Comment by Wholloran** [comment]
All:
- the Join us team member card should push the user to the opportunities page

Desktop:
- The line on the sides of "our values" is missing
- The team member images are too large, they should follow the margins of the three images above (desktop)
- have the "meet our team" button jump slightly less, currently, the meet the team header gets cut off when it jumps
- the Join us team member card should push the user to the opportunities page

Mobile:
- The "join us" card doesn't show the hover state text by default 

Tablet: 
- The last two team members\ (Alberto and Join us are centered but should be left-aligned 
- Our values icons are too large (they expand out further than they should. https://marvelapp.com/57djie4/screen/60484333

**Comment by Fdom92** [comment]
Added fix for the following:

- the Join us team member card should push the user to the opportunities page
- The line on the sides of "our values" is missing
- The team member images are too large, they should follow the margins of the three images above (desktop)
- have the "meet our team" button jump slightly less, currently, the meet the team header gets cut off when it jumps
- the Join us team member card should push the user to the opportunities page
- The last two team members\ (Alberto and Join us are centered but should be left-aligned
- Our values icons are too large (they expand out further than they should.

File: src/pages/app-team-landing/app-team-landing.tsx [hunk 1]
```diff
@@ -14,69 +14,60 @@ export class AppTeamLanding {
 
   @State() chips;
   data = {
+    'jedi-weller': {
```
Comment (Fdom92) [inline src/pages/app-team-landing/app-team-landing.tsx]: Change this to `jedidiah-weller` in order to fix other issue

**Comment by Wholloran** [comment]
- On a tablet, there is a little bit of white space above the hero image 
- When navigating to Jared's page from another team member's page, his headshot/background image doesn't always update. It happens about 70% of the time.
- Fernando's & Luis's headshot doesn't load on mobile (safari & Chrome)
- While on a team members page the other team members photos at the bottom are too large (using the wrong margins) need to apply the same change we did to the images on the same page
- Update Claudio's bio, (I had may make a small tweak to the language)

**Comment by luiskcs89** [comment]
Ready for QA

**Comment by Wholloran** [comment]
QA Passes

**Pull request merged**